packages/admin-stories/src/docs/components/Stack/Stack.stories.tsx

```diff
@@ -0,0 +1,283 @@
+                    <StackPage name="page2">
+                        <Page2 />
+                    </StackPage>
+                </StackSwitch>
+            </Stack>
+        );
+
+        function Page1() {
+            const switchApi = useStackSwitchApi();
+            return (
+                <>
+                    <h3>Page 1</h3>
+                    <button
+                        onClick={(e) => {
+                            switchApi.activatePage("page2", "test");
+                        }}
+                    >
+                        activate page2
+                    </button>
+                </>
+            );
+        }
+
+        function Page2() {
+            return (
+                <>
+                    <StackSwitch>
+                        <StackPage name="page2-1">
+                            <Page2_1 />
+                        </StackPage>
+                        <StackPage name="page2-2">page 2-2</StackPage>
+                    </StackSwitch>
+                </>
+            );
+        }
+
+        function Page2_1() {
+            const switchApi = useStackSwitchApi();
+            return (
+                <>
+                    <h3>Page 2-1</h3>
+                    <button
+                        onClick={(e) => {
+                            switchApi.activatePage("page2-2", "test");
+                        }}
+                    >
+                        activate page2-2
+                    </button>
+                </>
+            );
+        }
+    })
+    .add("DynamicTitle", () => {
+        return (
+            <Stack topLevelTitle="Example Stack">
+                <Toolbar>
+                    <ToolbarBackButton />
+                    <ToolbarBreadcrumbs />
+                </Toolbar>
+                <StackSwitch>
+                    <StackPage name="page1">
+                        <Foo payload="test1" />
+                        <Foo payload="test2" />
+                    </StackPage>
+                    <StackPage name="page2">
+                        {(payload) => (
+                            <StackPageTitle title={payload}>
+                                <div>Passed payload: {payload}</div>
+                            </StackPageTitle>
+                        )}
+                    </StackPage>
+                </StackSwitch>
+            </Stack>
+        );
+
+        function Foo({ payload }: { payload: string }) {
+            const switchApi = useStackSwitchApi();
+            return (
+                <>
+                    <button
+                        onClick={(e) => {
+                            switchApi.activatePage("page2", payload);
+                        }}
+                    >
+                        activate page2 with payload {payload}
+                    </button>
+                </>
+            );
+        }
+    })
+    .add("useStackSwitch", () => {
+        const [StackSwitch, api] = useStackSwitch();
```

Comment:
Since `switchApi` is used in all the other examples

```suggestion
        const [StackSwitch, switchApi] = useStackSwitch();
```